expat/expat_config.h.cmake

```diff
@@ -94,6 +94,17 @@
 /* Define to make XML Namespaces functionality available. */
 #cmakedefine XML_NS
 
+#if ! defined(_WIN32)
+/* Define to extract entropy from /dev/urandom. */
+#cmakedefine XML_DEV_URANDOM
+#endif
+
+/* Define to use UTF-16 chars. */
+#cmakedefine XML_UNICODE
+
+/* Define to use wchar_t as UTF-16 char type. */

```

Comment:
"rather than unsigned short" would be a cool bonus to mention here as well in my eyes